e2e/steps_file.js

```diff
@@ -185,6 +185,20 @@ module.exports = function () {
       }
     },
 
+    seeOrganisationInTab(pathToField, fieldValue) {
+      let path = [].concat(pathToField);
+      let fieldName = path.splice(-1, 1)[0];
+      const fieldSelector = `//mat-tab-body//*[@class="complex-panel" and .//*[@class="complex-panel-title" and .//*[text()="${path[0]}"]]]//*[@class="complex-panel" and .//*[@class="complex-panel-title" and .//*[text()="${path[1]}"]]]//*[contains(@class,"complex-panel-compound-field") and ..//*[text()="${fieldName}:"]]`;
```

Comment:
I think this part is a bit too strict, you could build it up like we do in the `tabFieldSelector` method, i.e. append to the end of the selector for each part of the path

Reply:
will do :)